README.md

```diff
@@ -95,7 +94,11 @@ The API is based on the [FastAPI](https://fastapi.tiangolo.com/) library.
   
       * `poetry init` # It will set up your local environment and repository details
       * `poetry env use python` # To find the virtual environment directory, created by poetry
-      * `source ~/ht-full-text-search-TUsF9qpC-py3.11/bin/activate` # Activate the virtual environment
+      * `source ~/ht-full-text-search-TUsF9qpC-py3.11/bin/activate` # Activate the virtual environment in Mac
```

Comment:
Is this the same as what `poetry env use python` does? Is there a need to find the `activate` script it produces directly rather than use poetry? I feel like maybe I asked this before -- that is, whether it's necessary to go manually go through the poetry cache to activate the virtualenv..